### PR TITLE
Feature/a11y audit

### PR DESCRIPTION
**Pull request by PanopticaRising**

### CODE REVIEW

**Comment by PanopticaRising** [comment]
This should be good eneough to make sure we address #45

File: src/Router.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { BrowserRouter, Switch, Route } from 'react-router-dom';
 import HomePage from './Home/HomePage';
 import VerificationLandingPage from './Login/VerificationLandingPage';
 import NavWrapper from './NavWrapper/NavWrapper';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (red-tt-bear) [inline src/Router.tsx:6]: We should comment out instead of ignoring the error

File: src/index.tsx [hunk 2]
```diff
@@ -4,6 +4,11 @@ import './index.css';
 import * as serviceWorker from './serviceWorker';
 import { Router } from './Router';
 
+if (process.env.NODE_ENV !== 'production') {
```
Comment (red-tt-bear) [inline src/index.tsx:7]: Is this guarenteed?

File: src/Router.tsx (hunk 1 above)
Comment (red-tt-bear) [reply src/Router.tsx:6]: @PanopticaRising said that if a file isn't included it throws errors which is why it is here
We should leave a comment for this
Better yet we could just do `import './Adviser/AdviserPage';`

File: src/index.tsx (hunk 2 above)
Comment (doombot-exe) [reply src/index.tsx:7]: yes. 
https://create-react-app.dev/docs/adding-custom-environment-variables/

**Pull request merged**